drivers/gles2/shader_gles2.h

```diff
@@ -320,19 +320,20 @@ class ShaderGLES2 {
 
 			} break;
 
-			case ShaderLanguage::TYPE_SAMPLER2D: {
-
-			} break;
-
-			case ShaderLanguage::TYPE_ISAMPLER2D: {
-
-			} break;
-
-			case ShaderLanguage::TYPE_USAMPLER2D: {
-
+			case ShaderLanguage::TYPE_SAMPLER2D:
+			case ShaderLanguage::TYPE_ISAMPLER2D:
+			case ShaderLanguage::TYPE_USAMPLER2D:
+			case ShaderLanguage::TYPE_SAMPLER2DARRAY:
+			case ShaderLanguage::TYPE_ISAMPLER2DARRAY:
+			case ShaderLanguage::TYPE_USAMPLER2DARRAY:
+			case ShaderLanguage::TYPE_SAMPLER3D:
+			case ShaderLanguage::TYPE_ISAMPLER3D:
+			case ShaderLanguage::TYPE_USAMPLER3D:
+			case ShaderLanguage::TYPE_SAMPLERCUBE: {
+				// FIXME: Implement those
 			} break;
 
-			case ShaderLanguage::TYPE_SAMPLERCUBE: {
+			case ShaderLanguage::TYPE_VOID: {
```

Comment:
```
drivers/gles2/shader_gles2.h:226:10: warning: enumeration value 'TYPE_ISAMPLER2DARRAY' not handled in switch [-Wswitch]
drivers/gles2/shader_gles2.h:226:10: warning: enumeration value 'TYPE_ISAMPLER3D' not handled in switch [-Wswitch]
drivers/gles2/shader_gles2.h:226:10: warning: enumeration value 'TYPE_SAMPLER2DARRAY' not handled in switch [-Wswitch]
drivers/gles2/shader_gles2.h:226:10: warning: enumeration value 'TYPE_SAMPLER3D' not handled in switch [-Wswitch]
drivers/gles2/shader_gles2.h:226:10: warning: enumeration value 'TYPE_USAMPLER2DARRAY' not handled in switch [-Wswitch]
drivers/gles2/shader_gles2.h:226:10: warning: enumeration value 'TYPE_USAMPLER3D' not handled in switch [-Wswitch]
drivers/gles2/shader_gles2.h:226:10: warning: enumeration value 'TYPE_VOID' not handled in switch [-Wswitch]
```
@karroffel @reduz

Reply:
After discussing with @reduz, `sampler3D` and 2D arrays are not supported in gles2.

Reply:
Fixed in new commit.